src/DetailsView/components/issues-table.tsx

```diff
@@ -125,6 +126,7 @@ export class IssuesTable extends React.Component<IssuesTableProps, IssuesTableSt
             <ExportDialog
                 deps={this.props.deps}
                 isOpen={this.state.isExportDialogOpen}
+                fileNameBase="AutomatedChecksReport"
```

Comment:
It looks like when I converged both reports on Export Dialog as part of the OF6 update, I [didn't wire up](https://github.com/Microsoft/accessibility-insights-web/commit/a1bc41e5b6247a30b16d634af8e04ec3cf6a3fc3#diff-0d3bccd06335566e1695cedac734218cL158
) the generated file name for the automated checks report. That's actually still available here at `this.state.exportName`. It contains the date (not the time) from the report and the target page name. There might be something reusable here.

Reply:
I believe I have wired it up as you expected. Let me know if it isn't right.

---
In reply to: [267592312](https://github.com/Microsoft/accessibility-insights-web/pull/398#discussion_r267592312) [](ancestors = 267592312)